src/frequenz/sdk/timeseries/_resampling/_config.py

```diff
@@ -134,7 +114,9 @@ class ResamplerConfig:
         passed to the resampling function.
     """
 
-    resampling_function: ResamplingFunction = average
+    resampling_function: ResamplingFunction = lambda samples, _, __: statistics.fmean(
+        [s[1] for s in samples]
```

Comment:
according to the docs this can be an iterable: https://docs.python.org/3/library/statistics.html#statistics.fmean

```suggestion
        s[1] for s in samples